doc/SLSKPROTOCOL.md

```diff
@@ -2290,21 +2290,23 @@ a peer. In Nicotine+, these messages are defined in slskmessages.py.
 
 1.  User A sends [ConnectToPeer](#server-code-18) to the Server with a unique
     token (indirect connection request)
-2.  User A sends a [PeerInit](#peer-init-code-1) to User B (direct connection
-    request)
-3.  The Server sends a [ConnectToPeer](#server-code-18) response to User B with
+2.  User A sends [GetPeerAddress](#server-code-3) to the Server with the same token and connection type
```

Comment:
GetPeerAddress doesn't take a token or connection type. You could mention the token in step 4 instead, and explain why we send GetPeerAddress here.